install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -53,7 +53,6 @@ spec:
           imagePullPolicy: {{ .Values.global.imagePullPolicy }}
           args:
           - "discovery"
-          - --registries={{ .Values.registries | join "," }}
```

Comment:
Do we want all configs come from MCP only?

Reply:
The registries is going to be deprecated and MCP should be a default. Also we are moving the registry to the configMap so we can add it to the API.

Reply:
This is a huge change, should doc in release note

Reply:
May influence compatibility